scripts/libuvc_installation.sh

```diff
@@ -31,7 +31,7 @@ then
 fi
 
 echo Installing Librealsense-required dev packages
-sudo apt-get install git libssl-dev libusb-1.0-0-dev pkg-config libgtk-3-dev unzip -y
+sudo apt-get install git cmake libssl-dev freeglut3-devcd libusb-1.0-0-dev pkg-config libgtk-3-dev unzip -y
```

Comment:
Is this a type @dorodnic? `freeglut3-dev**cd**`?

Reply:
Yes, thank you